### PR TITLE
3.2.0 Adaptation

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Adapted 3.2.0 CK2

Note for testers: Only bug fixes left here. Other events and decisions from 3.2.0 are commented out.

### CODE REVIEW

File: common/wonders/00_wonders.txt [hunk 1]
```diff
@@ -2067,15 +2067,21 @@ wonder_garden = {
...
-		evil_public_religion_trigger = no
-		
-		# Warcraft
-		is_tribal = no
-		
 		is_nomadic = no
-
-		# Warcraft
+		OR = {
+			religion_group = christian
```
Comment (MemeWizard) [inline common/wonders/00_wonders.txt]: So, you can't now build university/library?

File: common/wonders/00_wonders.txt [hunk 2]
```diff
@@ -1873,9 +1873,9 @@ wonder_palace = {
 		restore_cost_ticking = 2
 		upgrade_slots = 2
 		local_modifier = {
-			retinuesize = 250
 		}
 		owner_modifier = {
+			retinuesize = 250
```
Comment (MemeWizard) [inline common/wonders/00_wonders.txt]: update necropolis

File: common/wonders/00_wonders.txt (hunk 1 above)
Comment (zumbak04) [reply common/wonders/00_wonders.txt]: Fixed!

File: common/wonders/00_wonders.txt (hunk 2 above)
Comment (zumbak04) [reply common/wonders/00_wonders.txt]: Updated!

**Pull request merged**